packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -477,6 +477,12 @@ Different Policy types control settings for different operations. All Policy typ
 
 [OAuth Authorization Policy](/docs/reference/api/authorization-servers/#policy-object)
 
+[App Sign On Policy](#app-sign-on-policy) <ApiLifecycle access="ie" /><br>
```

Comment:
```suggestion
[App Sign-On Policy](#app-sign-on-policy) <ApiLifecycle access="ie" /><br>
```

Reply:
Please ensure that this is hyphenated in every instance.

Reply:
I will change it if I'm making another change nearby, but please do a find+replace to get the other instances that I don't change.